packages/uniforms/__tests__/AutoForm.tsx

```diff
@@ -27,66 +31,84 @@ describe('AutoForm', () => {
     validator.mockClear();
   });
 
-  describe('when changed', () => {
+  describe('when changes', () => {
     it('updates', () => {
-      // FIXME: AutoForm is not a valid Component.
-      const wrapper = mount<AutoForm | any>(
+      render(
         <AutoForm onChange={onChange} schema={schema} />,
+        { schema: { type: SimpleSchema2Bridge } },
+        { onChange },
       );
-
-      wrapper.instance().getContext().onChange('a', '2');
-
-      expect(onChange).toHaveBeenCalledTimes(1);
-      expect(onChange).toHaveBeenLastCalledWith('a', '2');
     });
-
     it('validates', () => {
       // FIXME: AutoForm is not a valid Component.
-      const wrapper = mount<AutoForm | any>(
-        <AutoForm onChange={onChange} schema={schema} />,
+      render(
+        <AutoForm
+          // TODO: delete ts-expect-error if this issue is resolved https://github.com/vazco/uniforms/issues/1165
+          // @ts-expect-error
```

Comment:
`@ts-expect-error` allows for adding comments and will cause `tsc` fail when no longer needed, so I propose this instead:
```suggestion
          // @ts-expect-error https://github.com/vazco/uniforms/issues/1165
```

Reply:
fixed